dotcom-rendering/src/web/lib/LiveBlogRenderer.tsx

```diff
@@ -70,6 +77,25 @@ export const LiveBlogRenderer = ({
 					</PinnedPost>
 				</>
 			)}
+			{switches.keyEventsCarousel && keyEvents?.length ? (
```

Comment:
At the risk of distracting, would it be possible to 
- move the Pinned Post out of the LiveBlogRenderer and into the LiveLayout
- move LiveBlogRenderer out of ArticleBody and into LiveLayout and remove ArticleBody from LiveLayout

Feels like it would be simpler to have everything in one place. Now that we've moved where the key events are displayed using article body doesn't make as much sense as it would now contain several non body things like pinned post and key events. But if this is too big a task it's fine, just a suggestion. 

If we do this then the KeyEventsCarousel would also only live in LiveLayout and not inside LiveBlogRenderer.